lib/desired-caps.js

```diff
@@ -0,0 +1,87 @@
+const desiredCapConstraints = {
+  platformName: {
+    presence: true,
+    isString: true,
+    inclusion: ['iOS']
+  },
+  browserName: {
+    isString: true,
+    inclusion: ['Safari']
+  },
+  app: {
```

Comment:
we'll also probably want to override the validating function to perform meta-cap validation, e.g., that either `bundleId` or `app` must exist